docs/skopeo.1.md

```diff
@@ -18,6 +18,10 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   default read from `$HOME/.kube/config`, which is set e.g. using
   `(oc login)`.
```

Comment:
(Above, for `atomic:`. The above needs updating, now it is `atomic:`hostname`/`namespace`/`stream`:`tag , and a better description would be “an image served by an Atomic Registry server” or something like that to suggest that the format is comparable with `docker://…`; current project and instance are explicitly contained in the above, but the user still needs to be logged into that Atomic Registry server.